integrationtest/mapper/orm-coordination-outbox-polling/src/test/java/org/hibernate/search/integrationtest/mapper/orm/coordination/outboxpolling/schema/OutboxPollingCustomEntityMappingIT.java

```diff
@@ -384,6 +393,82 @@ public void validMappingWithCustomFailingUuidGenerator() {
 				);
 	}
 
+	@Test
+	public void validMappingWithDefaultPayloadType() {
+		testPayloadType( PayloadType.LONG32VARBINARY );
+	}
```

Comment:
I'd add a test that *really* uses the default type, i.e. it doesn't set the configuration property at all.